src/main/java/com/google/sps/models/App.java

```diff
@@ -2,16 +2,65 @@
 import java.util.ArrayList;
 
 public class App {
-    int id;
-    String icon, name, category;
-    double rating, price;
-
-    App(int id, String name, String icon, String category, double rating, double price) {
-        this.id = id;
-        this.icon = icon;
-        this.name = name;
-        this.category = category;
-        this.rating = rating;
-        this.price = price;
```

Comment:
I remember you had compile issues earlier. Does this work if we remove it?

Reply:
it doesn't work if it's removed now, it still needs them in both app and builder